### PR TITLE
Some Corrections to Proptypes to Avoid Warnings

### PR DESCRIPTION
**Pull request by SudharakaP**
I observed the following two warnings when going in to a collective page; 

```
Warning: Failed prop type: Invalid prop `contributor.type` of value `EVENT` supplied to `ContributorAvatar`, expected one of ["USER","COLLECTIVE","FUND","ORGANIZATION","CHAPTER","ANONYMOUS"].

```
```
Warning: Failed prop type: Invalid prop `contributor.type` of value `EVENT` supplied to `ContributorCard`, expected one of ["USER","COLLECTIVE","FUND","ORGANIZATION","CHAPTER","ANONYMOUS"].
```

This PR fixes them. 😄

### CODE REVIEW

File: components/ContributorCard.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ ContributorCard.propTypes = {
     description: PropTypes.string,
     collectiveSlug: PropTypes.string,
     isIncognito: PropTypes.bool,
-    type: PropTypes.oneOf(['USER', 'COLLECTIVE', 'ORGANIZATION', 'FUND', 'CHAPTER', 'ANONYMOUS']),
+    type: PropTypes.oneOf(['USER', 'COLLECTIVE', 'ORGANIZATION', 'EVENT', 'FUND', 'CHAPTER', 'ANONYMOUS']),
```
Comment (znarf) [inline components/ContributorCard.js:191]: Maybe you can read from constants CollectiveType instead.

**Comment by znarf** [comment]
You should perhaps consider line 76 of Avatar.js too.

File: components/ContributorCard.js (hunk 1 above)
Comment (SudharakaP) [reply components/ContributorCard.js]: @znarf : Thanks. I've corrected that. Note that the `CollectiveType` doesn't contain, `CHAPTER` and `ANONYMOUS`; my understanding is that we don't use these anymore. Correct me if I am wrong here? 🤔

**Comment by SudharakaP** [comment]
> You should perhaps consider line 76 of Avatar.js too.

I've changed that as well. Thanks for pointing it out. 👍🏽

**Review by znarf — APPROVED** [review]

**Pull request merged**